lib/index.js

```diff
@@ -1,88 +1,356 @@
 'use babel';
 
+/* eslint no-use-before-define:0 */
+
 // eslint-disable-next-line import/extensions, import/no-extraneous-dependencies
 import { CompositeDisposable } from 'atom';
-import { dirname } from 'path';
-
-const badCommands = new Set();
+import { dirname, join } from 'path';
+import * as Helpers from 'atom-linter';
 
-const startMeasure = (baseName) => {
-  const startMark = `${baseName}-start`;
-  // Clear start mark from previous execution for the same file
-  if (performance.getEntriesByName(startMark).length) {
-    performance.clearMarks(startMark);
-  }
-  performance.mark(startMark);
+const configurationFile = '.codeclimate.yml';
```

Comment:
This should just be declared directly where it is being used, if later on this becomes something that needs configuring it can be changed then.